libs/core/include/core/byte_array/encoders.hpp

```diff
@@ -24,6 +24,7 @@ namespace byte_array {
 
 ConstByteArray ToBase64(ConstByteArray const &str);
 ConstByteArray ToHex(ConstByteArray const &str);
+ConstByteArray PartialHex(ConstByteArray const &str);
```

Comment:
This doesn't appear to be used anywhere in the code base remove?

Reply:
Yes, needs to be removed.